capabilities.md

```diff
@@ -157,6 +157,34 @@ Relic.
 
   An event batch is a [batch](#batches) that contains only events.
 
+## Logs
+
+The Telemetry SDK provides an API for sending log entries to
+New Relic's Log API. This implementation is _not recommended_ for sending high-throughput
+logs to New Relic. That capability should be provided by a log forwarder, rather than using
+the Telemetry SDK. The use-cases for using the implementation of the Log API in the Telemetry
```

Comment:
This sentence kind of goes around in a circle.  
I think I'd strike it entirely.

Reply:
which sentence, exactly?